src/xenia/kernel/session_object_json.cpp

```diff
@@ -0,0 +1,158 @@
+/**
+ ******************************************************************************
+ * Xenia : Xbox 360 Emulator Research Project                                 *
+ ******************************************************************************
+ * Copyright 2024 Xenia Emulator. All rights reserved.                        *
+ * Released under the BSD license - see LICENSE in the root for more details. *
+ ******************************************************************************
+ */
+
+#include <string>
+
+#include "xenia/kernel/session_object_json.h"
+
+namespace xe {
+namespace kernel {
+SessionObjectJSON::SessionObjectJSON()
+    : _sessionid(""),
```

Comment:
I would remove all these underscores. It's not convention used in Xenia if you really need them add them at the end, but iirc it should be used only for private variables in class